yaml/com/ibm/PLDM/ChapData.interface.yaml

```diff
@@ -0,0 +1,15 @@
+description: >
+    Implement this interface to pass Chap encrypted string to Host from PLDM.
+
+    com.ibm.PLDM.ChapData object implements this interface.
+
+properties:
+    - name: ChapName
+      type: string
+      description: >
+          ChapName is name associated with each secret chap password.
+
+    - name: ChapSecret
+      type: string
+      description: >
+          ChapSecret is encrypted password to transfer to Host.
```

Comment:
"ChapSecret is encrypted password to transfer to Host." --> "ChapSecret is an encrypted secret password transferred to the Host for the respective ChapName."